.github/workflows/add-review-team.yml

```diff
@@ -0,0 +1,31 @@
+  name: Label PRs
+
+  on:
+    issue_comment:
+      types: [created]
+
+  jobs:
+    label-pr:
+      name: Label PR
+      if: ${{ github.event.issue.pull_request }}
+      runs-on: ubuntu-latest
+      steps:
+        - name: team-label
+          uses: actions/github-script@v6
+          with:
+            script: |
+              const teams = ['@conda-forge/staged-recipes', '@conda-forge/help-python', '@conda-forge/help-python-c', '@conda-forge/help-r', '@conda-forge/help-java', '@conda-forge/help-nodejs', '@conda-forge/help-c-cpp', '@conda-forge/help-perl', '@conda-forge/help-julia', '@conda-forge/help-ruby'];
+
+              for (const team of teams) {
+                  let text = context.payload.comment.body;
+                  let result = text.includes(team);
+                  if (result) {
+                      label = team.replace("help-", "");
+                      github.rest.issues.addLabels({
+                          issue_number: context.issue.number,
+                          owner: context.repo.owner,
+                          repo: context.repo.repo,
+                          labels: [label, 'review-requested']
+                      })
+                  }
+              }
```

Comment:
We could additionally run Grayskull on the package and output that in a comment. Would make reviewing easier but adds more noise.